master/internal/audit_test.go

```diff
@@ -88,7 +88,7 @@ func TestAuditLogMiddleware(t *testing.T) {
 		require.NoError(t, err)
 		defer resp.Body.Close() //nolint:errcheck
 	}
-	// Then webui static files are ignored.
+	// Then frontend static files are ignored.
```

Comment:
in general, refer to 'WebUI' as 'cluster frontend'

from within the cluster code, though, just 'frontend'